rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -379,6 +380,21 @@ upgrade_masterfiles() {
 # Check and update config
 ################################################################################
 
+upgrade_conf_file() {
+
+  STEP="Upgrade /opt/rudder/etc/rudder-web.properties"
+  CONF_FILE="/opt/rudder/etc/rudder-web.properties"
+
+  # - 7.0.0~alpha1 : Rename property rudder.community.port into rudder.policy.distribution.port.cfengine
+  OLD_PROPERTY_NAME="rudder.community.port"
+  NEW_PROPERTY_NAME="rudder.policy.distribution.port.cfengine"
+  ATTRIBUTESET=$(grep -c "^${NEW_PROPERTY_NAME}[ \t]*=" ${CONF_FILE} || true)
+  if [ ${ATTRIBUTESET} -eq 0 ]; then
+    sed -e "s/$PROPERTY_NAME/rudder.policy.distribution.port.cfengine/" ${CONF_FILE} | grep -C 2 "port.cfengine"
```

Comment:
I don't understand this line

Reply:
That's... The test line, not the one that should be here.